docs/config/config.md

```diff
@@ -117,6 +117,7 @@ Provider represent a single provider from manifest
   | TenancyOcid       | string | OCID of the tenancy                                        |
   | OciFingerprint    | string | Fingerprint of the private key saved in `Credentials`      |
   | CompartmentOcid   | string | OCID of the compartment                                    |
+  | AccessKey         | string | AWS access key                                             |
```

Comment:
Isn't  AWS Secret key missing from here ?

Reply:
No, for the secret key, we are using the `Credentials` field, as mentioned in
```md
| OciFingerprint    | string | Fingerprint of the private key saved in `Credentials` 
```